media/transcoder/api/Transcoder.Samples/CreateJobWithConcatenatedInputs.cs

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START transcoder_create_job_with_concatenated_inputs]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Video.Transcoder.V1;
+
+public class CreateJobWithConcatenatedInputsSample
+{
+    public Job CreateJobWithConcatenatedInputs(
+        string projectId, string location, string inputUri1, decimal startTimeOffset1, decimal endTimeOffset1, string inputUri2, decimal startTimeOffset2, decimal endTimeOffset2, string outputUri)
+    {
+
+        decimal startTimeOffset1NanoSec = 1000000000 * decimal.Subtract(startTimeOffset1, decimal.Truncate(startTimeOffset1));
```

Comment:
There's no need to do any of this. Use `Duration.FromTimeSpan` to create the durations - which you can do inline later when creating the atoms.

Reply:
Done